Jenkinsfile

```diff
@@ -42,6 +42,7 @@ pipeline {
                 }
             }
             steps {
+                sh 'build/run make -j\$(nproc) vendor.check'
```

Comment:
if this fails, the main build step will not run, right? could we put the build first so contributors will see build failures before worrying about the vendoring?

Reply:
yes if this fails, the `build.all` would not run. I think this would this should push folks to run make vendor and then do a proper build. the ordering seems right to me. Why would a build on inconsistent dependencies matter?

Reply:
also with 727f46d I expect little to no vendor changes as people run `make vendor`.